routes/index.js

```diff
@@ -196,39 +196,73 @@ module.exports = function (passport) {
   });
 
 
-  router.get('/userManagement/:badge', checkAuth, function(req,res){
+  router.post('/userManagement/promoteUser/:badge',checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var BagdeID = req.params.badge;
+    dbAPI.modifyUser(BagdeID, { status: "Manager" }).then(function (result) {
```

Comment:
Before modifying this should check first that:
1. The user we are modifying actually has the status 'User'
2. Is also not an 'Admin'